cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -193,13 +193,20 @@ pub mod ump_constants {
 
 /// Trait for selecting the next core to build the candidate for.
 pub trait SelectCore {
+	fn select_core() -> (CoreSelector, ClaimQueueOffset);
 	fn select_core_for_child() -> (CoreSelector, ClaimQueueOffset);
```

Comment:
`selected_core` sounds better to me, and now `select_core_for_child` -> `select_next_core`  also better .